.travis.yml

```diff
@@ -24,10 +24,26 @@ addons:
     - libseccomp-dev
     - wget
     - pkg-config
+    - libopenmpi-dev
 install:
+# create and activate conda environment for testing
+- if [[ "$TRAVIS_PYTHON_VERSION" == "2.7" ]]; then
+    wget https://repo.continuum.io/miniconda/Miniconda2-latest-Linux-x86_64.sh -O miniconda.sh;
+  else
+    wget https://repo.continuum.io/miniconda/Miniconda3-latest-Linux-x86_64.sh -O miniconda.sh;
+  fi
+- bash miniconda.sh -b -p $HOME/miniconda
+- source "$HOME/miniconda/etc/profile.d/conda.sh"
+- hash -r
+- conda config --set always_yes yes --set changeps1 no
+- conda update -q conda
+- conda create -q -n test-environment python=$TRAVIS_PYTHON_VERSION
+- conda activate test-environment
+
 - pip install codecov
 - pip install -r requirements.txt
 - pip install -r test_requirements.txt
+- conda install -c conda-forge petsc4py
```

Comment:
just curious, do the pip install instructions not work, or this just brings mpicc along?

Reply:
I haven't tried too hard, but had some trouble doing `pip install petsc petsc4py` as it says on pypi. The conda recipe seems really nice, though, and the repo technically works without it in `requirements.txt`.